profiles/databricks.yaml

```diff
@@ -14,5 +14,14 @@
 INGEST_DATA_LOCATION: dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled
 # Specify the format of the dataset
 INGEST_DATA_FORMAT: spark_sql
+# Use a section of the TLC Trip Record Dataset for model development
+# (https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
+INGEST_SCORING_DATA_LOCATION: dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled
+# Specify the format of the dataset
+INGEST_SCORING_DATA_FORMAT: spark_sql
 # Override the default train / validation / test dataset split ratios
 SPLIT_RATIOS: [0.75, 0.125, 0.125]
+# Specify the output location of the batch scoring predict step
+OUTPUT_DATA_LOCATION: "dbfs:/temp/batch_scoring/sample_output.parquet"
+# Specify the output format of the batch scoring predict step
+OUTPUT_DATA_FORMAT: parquet
```

Comment:
Is this profile tested? Shouldn't them be `SCORED_OUTPUT_DATA_LOCATION` and `SCORED_OUTPUT_DATA_FORMAT`?

Reply:
That's a bug on my part....yeah i dont think this profile gets tested in unit tests which is all i was using.  Will fix.